### PR TITLE
Frontend dependencies audit

### PR DESCRIPTION
**Pull request by glebcha**
## Description
This PR add frontend dependencies vulnerabilities audit both for current and preview ui.

## Additional context and related issues
[Original issue with proposal](https://github.com/trinodb/trino/issues/23437)

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: core/trino-web-ui/src/main/resources/webapp-preview/README.md [hunk 1]
```diff
@@ -1,43 +1,55 @@
+
```
Comment (wendigo) [inline core/trino-web-ui/src/main/resources/webapp-preview/README.md:1]: Revert.

Comment (glebcha) [reply core/trino-web-ui/src/main/resources/webapp-preview/README.md]: My fault, missed formatting.

Fixed issue.

**Review by ebyhr — COMMENTED** [review]
Could you fix CI failure? 
```
Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.1:npm (package (webapp)) on project trino-web-ui: Failed to run task: 'npm run package:clean' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
```

**Comment by glebcha** [comment]
> Could you fix CI failure? 
> ```
> Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.1:npm (package (webapp)) on project trino-web-ui: Failed to run task: 'npm run package:clean' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
> ```

Sure, will check!

**Comment by glebcha** [comment]
@ebyhr I tried to build locally with no issues, but had no luck with investigation of CI issue.
 
P.S.: After faulty force-push github closed this PR, so I had to open new https://github.com/trinodb/trino/pull/24216

**Pull request closed without merging**